src/LinqKit.Microsoft.EntityFrameworkCore/ExpandableDbContextOptionsBuilderExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿#if EFCORE3 || EFCORE5
+
+using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+
+namespace LinqKit
+{
+    /// <summary>
+    /// Replaces method calls with lambda expressions.
+    /// </summary>
+    public static class ExpandableDbContextOptionsBuilderExtensions
+    {
+        /// <summary>
+        /// Replaces method calls with lambda expressions.
+        /// </summary>
+        /// <param name="optionsBuilder">The builder being used to configure the context.</param>
+        /// <returns>The options builder so that further configuration can be chained.</returns>
+        public static DbContextOptionsBuilder WithExpressionExpanding(this DbContextOptionsBuilder optionsBuilder)
+        {
+            if (optionsBuilder is null)
+                throw new ArgumentNullException(nameof(optionsBuilder));

```

Comment:
surround with { and }

This is a general comment for some more code you've added.